index.html

```diff
@@ -101,6 +101,7 @@ <h1>Index of python-pkgs</h1>
     <a href="pytz-2017.2-py2.py3-none-any.whl">pytz-2017.2-py2.py3-none-any.whl</a><br/>
     <a href="requests-2.10.0-py2.py3-none-any.whl">requests-2.10.0-py2.py3-none-any.whl</a><br/>
     <a href="scandir-1.10.0-cp27-cp27m-macosx_10_15_x86_64.whl">scandir-1.10.0-cp27-cp27m-macosx_10_15_x86_64.whl</a><br/>
+    <a href="scandir-1.10.0-cp38-cp38-macosx_11_0_x86_64.whl">scandir-1.10.0-cp38-cp38-macosx_11_0_x86_64.whl</a><br/>
```

Comment:
Shouldnt there be a wheel for pathlib2? Did it not built one?

Reply:
it did not D: i was confused about that too but i think pathlib2 is `pathlib2-2.3.5-py2.py3-none-any.whl` so it doesn't care about the os version?